src/content/architecture/_index.en.md

```diff
@@ -5,3 +5,13 @@ pre: "<b>3. </b>"
 weight: 15
 ---
 
+The diagram below illustrates the basic architecture of Submariner:
+
+![Submariner Architecture](/images/submariner/architecture.jpg)
+
+Submariner consists of several main components that work in conjunction to securely connect workloads across multiple Kubernetes clusters, both on-premise and on public clouds. 
```

Comment:
Should we also mention Globalnet in here? I know that the intention was to capture the "main components" only, but if the reader follow the wizard (clicks the arrow on the right side), then the next page that shows up is the one describing Globalnet. It's probably not the flow we should aim for. Similarly, there is a dedicated page for Service Discovery. Perhaps we should mention what it is in here?

BTW, can we control the order of the pages under Architecture? We should present the pages in a more logical order.

Reply:
yeah we can mention the additional Globalnet and Service Discovery components separately here. I was outlining the components in the diagram.

I was wondering about the ordering too. I haven't figured out yet how that is determined - it is not alphabetical. Maybe @mangelajo knows?

Reply:
Haven't tested it, but the "weight" parameter should do the trick: https://themes.gohugo.io//theme/hugo-theme-learn/en/cont/pages/

Reply:
yup - thanks!  I re-ordered the pages:  Broker, Gateway Engine, Route Agent, Globalnet, Service Discovery,, Network.  Also listed Globalnet and Service Discovery on the main Architecture page,

Reply:
Looks great @tpantelis. As per my slack conversation with @mangelajo ans @sridhargaddam, it seems like we have no plans to work on the "Network" page any time soon so we better just removed it instead of showing a blank page. Do you want to do so as part of this PR?

Reply:
TBH I'm not sure what was intended to go in this section. Let's remove it.

Reply:
yes - fixed